python/indy-agent.py

```diff
@@ -52,75 +52,48 @@
 
 LOOP = asyncio.get_event_loop()
 
-AGENT = web.Application()
+WEBAPP = web.Application()
 
-aiohttp_jinja2.setup(AGENT, loader=jinja2.FileSystemLoader('view'))
+aiohttp_jinja2.setup(WEBAPP, loader=jinja2.FileSystemLoader('view'))
 
-AGENT['msg_router'] = Router()
-AGENT['msg_receiver'] = Receiver()
+WEBAPP['msg_router'] = Router()
+WEBAPP['msg_receiver'] = Receiver()
 
-AGENT['ui_event_queue'] = UIEventQueue(LOOP)
-AGENT['ui_router'] = Router()
+WEBAPP['ui_event_queue'] = UIEventQueue(LOOP)
+WEBAPP['ui_router'] = Router()
 
-AGENT['conn_router'] = Router()
-AGENT['conn_receiver'] = Receiver()
+WEBAPP['conn_router'] = Router()
+WEBAPP['conn_receiver'] = Receiver()
 
-AGENT['agent'] = Agent()
-AGENT['modules'] = {
-    'connection': Connection(AGENT['agent']),
-    'ui': Ui(AGENT['agent']),
-    'admin_walletconnection': AdminWalletConnection(AGENT['agent'])
+WEBAPP['agent'] = Agent()
+WEBAPP['modules'] = {
+    'connection': Connection(WEBAPP['agent']),
+    'ui': Ui(WEBAPP['agent']),
+    'admin_walletconnection': AdminWalletConnection(WEBAPP['agent'])
 }
-AGENT['agent'].modules = AGENT['modules']
+WEBAPP['agent'].modules = WEBAPP['modules']
 
 UI_TOKEN = uuid.uuid4().hex
-AGENT['agent'].ui_token = UI_TOKEN
+WEBAPP['agent'].ui_token = UI_TOKEN
 
 ROUTES = [
     web.get('/', modules.ui.root),
-    web.get('/ws', AGENT['ui_event_queue'].ws_handler),
+    web.get('/ws', WEBAPP['ui_event_queue'].ws_handler),
     web.static('/res', 'view/res'),
-    web.post('/indy', AGENT['msg_receiver'].handle_message),
-    web.post('/offer', AGENT['conn_receiver'].handle_message)
+    web.post('/indy', WEBAPP['msg_receiver'].handle_message),
+    web.post('/offer', WEBAPP['conn_receiver'].handle_message)
 ]
 
-AGENT.add_routes(ROUTES)
+WEBAPP.add_routes(ROUTES)
 
-RUNNER = web.AppRunner(AGENT)
+RUNNER = web.AppRunner(WEBAPP)
 LOOP.run_until_complete(RUNNER.setup())
 
 SERVER = web.TCPSite(runner=RUNNER, port=PORT)
 
-async def connect_wallet(agent_name, passphrase):
-    #set wallet name from msg contents
-
-    AGENT['agent'].owner = agent_name
-    wallet_name = '%s-wallet' % AGENT['agent'].owner
-
-    wallet_config = json.dumps({"id": wallet_name})
-    wallet_credentials = json.dumps({"key": passphrase})
-
-    # pylint: disable=bare-except
-    # TODO: better handle potential exceptions.
-    try:
-        await wallet.create_wallet(wallet_config, wallet_credentials)
-    except Exception as e:
-        print(e)
-
-    try:
-        AGENT['agent'].wallet_handle = await wallet.open_wallet(wallet_config,
-                                                       wallet_credentials)
-    except Exception as e:
-        print(e)
-        print("Could not open wallet!")
-    AGENT['agent'].initialized = True
-    (_, AGENT['agent'].endpoint_vk) = await did.create_and_store_my_did(
-        AGENT['agent'].wallet_handle, "{}")
-
-
-if AGENTINITINCLI:
+if WEBAPPINITINCLI:
```

Comment:
this part broke.... should be a quick fix.

Reply:
Thanks for the catch